packages/web/src/pages/upload-page/fields/SwitchRowField.tsx

```diff
@@ -34,8 +35,10 @@ export const SwitchRowField = (props: ToggleFieldProps) => {
   return (
     <div className={styles.root}>
       <div className={styles.content}>
-        <h3 className={cn(styles.title, styles.modalHeading)}>{header}</h3>
-        <p className={styles.description}>{description}</p>
+        <Text className={styles.title} variant='title' size='Large'>
```

Comment:
strange to me to have the variant lower and the size capitalized. Can we make them all accept lower?